app/extensions/brave/content/scripts/passwordManager.js

```diff
@@ -74,11 +74,19 @@ if (chrome.contentSettings.passwordManager == 'allow') {
             }, usernameElem.value || '')
         }
       })
+      usernameElem.addEventListener('blur', (e) => {
```

Comment:
this breaks autofill, i think because the username context menu is gone by the time the click event is fired.